client/my-sites/upgrades/checkout/credit-card-payment-box.jsx

```diff
@@ -33,17 +90,60 @@ var CreditCardPaymentBox = React.createClass( {
 		this.props.onToggle( 'paypal' );
 	},
 
-	content: function() {
-		var cart = this.props.cart;
+	progressBar: function() {
+		return (
+			<div className="credit-card-payment-box__progress-bar">
+				{ this.translate( 'Processing payment…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Error processing payment' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/checkout-loading-state)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).